homeassistant/components/octoprint/sensor.py

```diff
@@ -129,7 +127,7 @@ def update(self):
             self._state = self.api.update(
                 self.sensor_type, self.api_endpoint, self.api_group, self.api_tool
             )
-        except requests.exceptions.ConnectionError:
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Don't catch generic exceptions (hence that you had to disable the check!). Specify all exceptions that you can expect.

Reply:
I will revert that

Reply:
reverted in 3342fe2